float32 x
float32 y
float32 z
float32 yaw

float32 vel_x
float32 vel_y
